storybook/pages/CreateChannelPopupPage.qml

```diff
@@ -50,12 +50,12 @@ SplitView {
 
             onCreateCommunityChannel: (chName, chDescription, chEmoji, chColor, chCategoryId) => {
                 logs.logEvent("onCreateCommunityChannel",
-                              ["chName", "chDescription", "chEmoji", "chColor", "chCategoryId"], arguments)
+                              ["chName", "chDescription", "chEmoji", "chColor", "chCategoryId"], chName, chDescription, chEmoji, chColor, chCategoryId)
```

Comment:
It's still unresolved :) Could you please respond to my previous remark?

Reply:
done